akka-docs/rst/java/code/docs/testkit/ParentChildTest.java

```diff
@@ -45,7 +45,7 @@ public Receive createReceive() {
     public Receive createReceive() {
       return receiveBuilder()
         .matchEquals("ping", message -> {
-          getContext().parent().tell("pong", self());
+          getContext().getParent().tell("pong", self());
```

Comment:
I think it's good to have the `getParent` also but I think we should consistently use `parent()` in documentation examples to be consistent with what we have done with `self` vs `getSelf`.

The main reason for not removing `getSelf` is that it's widely used and I can understand that some users prefer the more classic `get` idiom.